src/formatDate.js

```diff
@@ -7,8 +7,56 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateElements = date.split(fromFormat[fromFormat.length - 1]);
+  const newFromDate = {};
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    const format = fromFormat[i];
+
+    if (format === 'YYYY') {
+      newFromDate.year = +dateElements[i];
+    } else if (format === 'YY') {
+      newFromDate.year = +dateElements[i];
+
+      if (newFromDate.year < 30) {
+        newFromDate.year += 2000;
+      } else {
+        newFromDate.year += 1900;
+      }
```

Comment:
The conversion of the year from `YY` to `YYYY` is correct, but it's better to perform this conversion when you are pushing the year into `formattedDate` in the second loop. This way, you avoid modifying the year twice if it's not necessary.